### PR TITLE
[Feature] Support `CENet` in `Projects`

### PR DESCRIPTION
**Pull request by Xiangxu-0103**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: projects/CENet/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+python tools/test.py projects/CENet/configs/cenet-64x512_4xb4_semantickitti.py ${CHECKPOINT_PATH}
+```
+
+## Results and models
+
+### NuScenes
+
+|                        Backbone                        | Input resolution | Mem (GB) | Inf time (fps) | mIoU  |         Download         |
+| :----------------------------------------------------: | :--------------: | :------: | :------------: | :---: | :----------------------: |
+| [CENet](./configs/cenet-64x512_4xb4_semantickitti.py)  |     64\*512      |          |                | 61.10 | [model](<>) \| [log](<>) |
```
Comment (sunjiahao1999) [inline projects/CENet/README.md:51]: report FPS.

File: projects/CENet/cenet/transforms_3d.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+from typing import Sequence
+
+import numpy as np
+from mmcv.transforms import BaseTransform
+
+from mmdet3d.registry import TRANSFORMS
+
+
+@TRANSFORMS.register_module()
+class SemkittiRangeView(BaseTransform):
```
Comment (sunjiahao1999) [inline projects/CENet/cenet/transforms_3d.py:11]: Can this be put in mmdet3d/models/data_preprocessors/data_preprocessor.py? Refer to voxelization

File: projects/CENet/configs/cenet-64x512_4xb4_semantickitti.py [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+# i.e. we consider the unlabeled class as the last one, which is different
+# from the original implementation of some methods e.g. Cylinder3D.
+dataset_type = 'SemanticKittiDataset'
+data_root = 'data/semantickitti/'
+class_names = [
+    'car', 'bicycle', 'motorcycle', 'truck', 'bus', 'person', 'bicyclist',
+    'motorcyclist', 'road', 'parking', 'sidewalk', 'other-ground', 'building',
+    'fence', 'vegetation', 'trunck', 'terrian', 'pole', 'traffic-sign'
+]
+labels_map = {
```
Comment (sunjiahao1999) [inline projects/CENet/configs/cenet-64x512_4xb4_semantickitti.py:16]: remove redundant config. These can inherit from _base_

File: projects/CENet/cenet/transforms_3d.py (hunk 2 above)
Comment (Xiangxu-0103) [reply projects/CENet/cenet/transforms_3d.py:11]: Maybe some other augmentations can be performed on the `range image` such as `range-image_shift`, so we suggest converting point cloud to range image in the `transforms` pipeline.

**Review by sunjiahao1999 — APPROVED** [review]

**Pull request merged**